src/weather-bar.ts

```diff
@@ -322,6 +347,7 @@ export class WeatherBar extends LitElement {
       grid-area: right;
       border: 1px solid var(--divider-color, lightgray);
       border-width: 0 0 0 1px;
+      border-collapse: collapse;
```

Comment:
This may lead to the axis marks being too narrow. Can we double check that the axis ticks remain the same after this change?